### PR TITLE
Switch to fixed rubygemsrecipe fork

### LINKED ISSUE #1543: Cannot build adhocracy
I got the following error when building adhocracy:

```
rubygems: Can't find latest rubygems version.
While:
  Installing rubygems.
Error: Configuration error
*********************************************
Overwriting meinberlin-requirements.txt
*********************************************
```

The rubygems is broken and scraps for URL on the ruby page which don't exist anymore:

https://bitbucket.org/sirex/rubygemsrecipe/src/a37d4859a342b9250aeca8977ed565514a00fee4/rubygems.py?at=default#rubygems.py-126

### PR DESCRIPTION
**Pull request by xi**
*fixes #1543*

Installing rubygems with buildout currently fails because ruby changed their API and rubygemsrecipe has not yet been updated.

I fixed the upstream issue and switched to the fixed fork in the pull request. If travis succeedes in this PR I will create an upstream pull request.

### CODE REVIEW

File: src/adhocracy_frontend/sources.cfg [hunk 1]
```diff
@@ -4,3 +4,5 @@ auto-checkout = *
 
 [sources]
 supervisor = git https://github.com/Supervisor/supervisor.git rev=03333b854625d5e948d077d1110f1b7f14bd2892
+rubygemsrecipe = hg ssh://hg@bitbucket.org/ximmm/rubygemsrecipe branch=fix-rubygems-url
```
Comment (nidico) [inline src/adhocracy_frontend/sources.cfg]: this should be `https`, not `ssh`.

**Comment by nidico** [comment]
This seems to work. The travis failures are other issues:

- sometimes the build breaks early, don't know why
- a jsunit test fails

**Comment by xi** [comment]
Upstream pull request is here: https://bitbucket.org/sirex/rubygemsrecipe/pull-requests/8/fix-rubygems-url/diff

**Comment by pallix** [comment]
thanks you very much @xi. Developers need to install mercurial on their machine.

**Comment by xi** [comment]
> Developers need to install mercurial on their machine

yeay! I guess we could also create a git clone or something like that. What should we do?

**Comment by nidico** [comment]
`sudo apt install mercurial` is a good choice.

This is only temporary until upstream merges the PR.

**Comment by pallix** [comment]
I agree with @nidico

**Pull request merged**